ci/check-style.sh

```diff
@@ -68,7 +68,7 @@ for file in $(find google/cloud -name '*.h' -o -name '*.cc' -print); do
   else
       mv -f "${file}.tmp" "${file}"
   fi
-done
+done < <(find google/cloud \( -name '*.h' -o -name '*.cc' \) -print0)
```

Comment:
I think I would just use a pipe rather than process substitution.  That is, ...
```
find google/cloud \( -name '*.h' -o -name '*.cc' \) -print0 |
  while IFS= read -r -d $'\0' file; do
    ...
  done
```
Then the `find` is closer to the `read`.